react-frontend/src/recognition-list-components/RecognitionCardComponent.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { Row, Col } from "core-components/grid/GridComponent";
+import { Card } from "core-components/card/CardComponent";
+import { BsCircle } from "core-components/icon/icons";
+import ImageComponent from "core-components/image/ImageComponent";
+import RecognitionCardHeaderComponent from "recognition-list-components/RecognitionCardHeaderComponent";
+import RecognitionTextComponent from "recognition-list-components/RecognitionTextComponent";
+import HighFiveComponent from "shared-components/high-five-components/HighFiveComponent";
+
+const giveHighFive = () => {
+  //to to for give high five
+};
+
+const ImgD = styled.div`
+  border-radius: 20px;
+  height: auto;
+  width: auto;
+  overflow: hidden;
+`;
+
+const ImgM = styled.div`
+  border-radius: 20px;
+  height: auto;
+  width: auto;
```

Comment:
1. Given that both height and width are auto, do we need to specify them?

2. Why overflow has been set to auto?